src/main/java/com/indeed/operators/rabbitmq/api/RabbitMQApiClient.java

```diff
@@ -136,6 +155,24 @@ public void createOrUpdateShovel(final RabbitMQConnectionInfo connectionInfo, fi
         httpClient.newCall(req).execute();
     }
 
+    public void deleteShovel(final RabbitMQConnectionInfo connectionInfo, final String vhost, final String shovelName) throws IOException {
+        final String url = String.format(
+                "%s/parameters/%s/%s/%s",
+                buildRootUrl(connectionInfo),
+                "shovel",
```

Comment:
Should this be a constant?

Reply:
probably

Reply:
but i'm going to leave it for now since this whole thing will be getting a massive refactor very shortly